src/common/wallet-state.ts

```diff
@@ -638,7 +638,8 @@ function useWalletState(initialState?: Partial<WalletStateParams>): WalletData {
     }
 
     const privateKey = getCurrentPrivateKey(password)
-    const tx = await web3.eth.accounts.signTransaction(txConfig, privateKey)
+
+    const tx = await web3.eth.accounts.signTransaction(txConfig, padPrivateKey(privateKey))
```

Comment:
Shouldn't we also pad the key as early as possible (e.g. at moment user restores the wallet) as well as when user wants to display the key? That would make it obsolete over time to add padding here as well as users would always see "normalized" keys. Also, it's more for @conqeror - in future it may be worth designating a type for a private key/address that both ensures invariants like proper length as well as allows wallet domain code to not care about checking them all the time, but take them for granted.